src/components/SocialSection.jsx

```diff
@@ -6,7 +6,7 @@ import { getSubjectLocation, getThumbnailSource } from '../lib/get-subject-locat
 const SocialSection = ({ project, subjectsOfNote }) =>
   <div className="home-page__social-section">
 
-    {subjectsOfNote && subjectsOfNote.length && (
+    {subjectsOfNote.length ? (
```

Comment:
Out of curiosity, why was the first part of `subjectsOfNote && subjectsOfNote.length` removed?

I've traced the code and it looks functional - subjectsOfNote is unlikely to be `undefined` unless there was a catastrophic reply from the API, so the code is still practically safe, but the lack of that existence check makes it just a little more brittle.

Reply:
This was because the numeral "0" appeared when there was no subjectOfNote being discussed (empty array) as is currently present on the main site. Adding a `false` at the end of this ternary seemed to take care of the problem.

Reply:
Ah, I understand why it's a ternary, it's that weird zero being rendered. Sorry, I should rephrase: why is it `(subjectsOfNote.length) ? something : false` instead of `(subjectsOfNote && subjectsOfNote.length) ? something : false` ?

Again, it's a curiosity question; it's no big deal as I traced the code and it should be safe.

Reply:
Ahh, good point there. I see what you mean, and having the extra check on `subjectsOfNote` definitely would help here. I think I experimented with a couple different ways of doing this and forgot to add back in the `subjectsOfNote && subjectsOfNote.length` once I found a working solution. I can add back in the extra check with a future PR.